spec/requests/cities_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe "Cities", type: :request do
+  let(:user) { create(:user, role_id: 6) }
```

Comment:
```suggestion
  let(:user) { create(:user_sysAdmin) }
```
Если возникает ошибка с созданием юзера, то эт наверн из-за того, что нужно добавить поле confimed_at в фактории юзера. Пример в таске L-42-measurement-units